content/uk/docs/reference/glossary/affinity.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Спорідненість
+id: affinity
```

Comment:
```suggestion
title: Affinity
id: affinity
```

Це складова назва маніфестів.

Reply:
I'd mention the English term inside the running text, so that readers can be clear on what is mean,  and also as an AKA so that people can find the term.